tools/editor/editor_file_dialog.cpp

```diff
@@ -1248,6 +1277,18 @@ EditorFileDialog::EditorFileDialog() {
 	mode=MODE_SAVE_FILE;
 	set_title(TTR("Save a File"));
 
+	ED_SHORTCUT("file_dialog/go_back", TTR("Go Back"), KEY_MASK_ALT|KEY_LEFT);
+	ED_SHORTCUT("file_dialog/go_fowrard", TTR("Go Forward"), KEY_MASK_ALT|KEY_RIGHT);
+	ED_SHORTCUT("file_dialog/go_up", TTR("Go Up"), KEY_MASK_ALT|KEY_UP);
+	ED_SHORTCUT("file_dialog/refresh", TTR("Refresh"), KEY_MASK_CMD|KEY_F5); // ctrl + f5 else it launches the game as well..
```

Comment:
Can't we prevent FileDialog from passing the event further, so that F5 does not launch the game? CC @neikeq, the event propagation expert :p